### PR TITLE
Extract view for a participant in a call

### PR DESCRIPTION
**Pull request by danxuliu**
Note that this pull request is a pure refactoring; there should be no behaviour changes (not even bug fixes).

### CODE REVIEW

File: js/views/videoview.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+				userName = userName || t('spreed', 'Guest');
+			}
+
+			this.getUI('nameIndicator').text(userName);
+		},
+
+	});
+
+	OCA.SpreedMe.Views.VideoView = VideoView;
```
Comment (nickvergessen) [inline js/views/videoview.js]: For new code please use `OCA.Talk`

File: js/views/videoview.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+			this.getUI('screenSharingIndicator').tooltip({
+				placement: 'top',
+				trigger: 'hover'
+			});
+		},
+
+		setUser: function(userId, userName) {
+			if (userId && userId.length) {
+				this.getUI('avatar').avatar(userId, 128);
```
Comment (nickvergessen) [inline js/views/videoview.js]: this should just be dropped I guess, or well only be done on the first call?

File: js/views/videoview.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+			} else {
+				this.getUI('avatar').imageplaceholder('?', userName, 128);
+				this.getUI('avatar').css('background-color', '#b9b9b9');
+
+				userName = userName || t('spreed', 'Guest');
+			}
+
+			this.getUI('nameIndicator').text(userName);
```
Comment (nickvergessen) [inline js/views/videoview.js]:
```
Uncaught TypeError: Cannot read property 'text' of undefined
    at n.setUser (videoview.js:88)
    at Object.add (webrtc.js:368)
    at webrtc.js:170
    at Array.forEach (<anonymous>)
    at usersChanged (webrtc.js:153)
    at usersInCallChanged (webrtc.js:255)
    at Function.<anonymous> (webrtc.js:289)
    at Internal.OCA.Talk.Signaling.Base._trigger (signaling.js:124)
    at Internal.<anonymous> (signaling.js:614)
    at Function.each (jquery.js:374)
```

Comment (nickvergessen) [reply js/views/videoview.js]: This is when trying to join a public call as the owner and as a guest.

**Comment by danxuliu** [comment]
Ready for review.

**Pull request merged**